purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -489,31 +519,10 @@ class Purchases @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE) intern
             packageToPurchase.product,
             packageToPurchase.offering,
             upgradeInfo,
-            listener.toProductChangeListener()
+            listener.toProductChangeCallback()
         )
     }
 
-    private fun MakePurchaseListener.toProductChangeListener(): ProductChangeListener {
```

Comment:
This has been removed because now they should be `ProductChangeCallbacks`. I added the conversion functions in `ProductChangeCallback`